wm/frame.go

```diff
@@ -221,12 +221,26 @@ func (f *frame) maximizeApply() {
 	f.client.restoreX = f.x
 	f.client.restoreY = f.y
 
-	maxWidth, maxHeight := desktop.Instance().ContentSizePixels()
+	var screenIndex, startX, startY int = 0, 0, 0
+	screens := f.client.wm.Screens()
+	if len(screens) > 1 {
```

Comment:
Shouldn't this be in a helpful function (like ScreenForWindow) somewhere like the Screens interface?

Reply:
Indeed. Added.